src/Octoshift/Models/AdoRepository.cs

```diff
@@ -0,0 +1,9 @@
+namespace Octoshift.Models;
+
+public record AdoRepository
+{
+    public string Id { get; init; }
+    public string Name { get; init; }
+    public uint Size { get; init; }
```

Comment:
1. Based on [ADO Repository List](https://docs.microsoft.com/en-us/rest/api/azure/devops/git/repositories/list?view=azure-devops-rest-7.1#gitrepository) documentation, `Size` should always be present, so I did not make it nullable (`unit?`).
2. The docs says that the `Size` is an integer but since we cannot have a negative repo size I made it a `uint`.